Sources/PulseUI/Features/Console/List/ConsoleListView.swift

```diff
@@ -174,7 +174,7 @@ private struct _ConsoleListView: View {
                     } else {
                         ConsoleListContentView(proxy: proxy)
                     }
-                }
+                }.scrollContentBackground(.hidden)
```

Comment:
This makes the view on macOS more beautiful in my opinion and also allows to configure an overall console background color using:

```swift
ConsoleView()
    .backgroundColor(Color.black)
```

For example.